composer/cicd_sample/README.md

```diff
@@ -0,0 +1,3 @@
+# Composer CI/CD Sample
```

Comment:
If you haven't already, is it worth mentioning the Airflow versions this tutorial/code sample is compatible with? I see the guide in the CL is under 'Cloud Composer 1 guide' but Composer version isn't explicitly mentioned

Reply:
Great point - I'll add this in the README and it'll be more clear in the accompanying docs. It should be compatible with all versions of Airflow and Composer.